lib/constants.go

```diff
@@ -911,9 +917,24 @@ func (params *DeSoParams) EnableRegtest() {
 	// Set the PoS default jail inactive validator grace period epochs to 3.
 	params.DefaultJailInactiveValidatorGracePeriodEpochs = 3
 
+	// Check for consensus transition every second.
+	params.FastHotStuffConsensusTransitionCheckDuration = 1 * time.Second
+
 	// In regtest, we start all the fork heights at zero. These can be adjusted
 	// for testing purposes to ensure that a transition does not cause issues.
 	params.ForkHeights = RegtestForkHeights
+	if isAcceleratedRegtest {
+		params.ForkHeights.ProofOfStake2ConsensusCutoverBlockHeight = 30
```

Comment:
So I actually had a manually-hacked version of this value, and whenever I set it to <40 I had issues so mine was always set to 40. If you've found that it works with 30 then leave it. But I think the epoch duration caused some issues under 40, and I suspect it was due to not having enough time to build the snapshot validator set or something (like you need two epochs or something).

Reply:
30 works b/c our epoch lengths are 5 blocks! you may have been testing w/ 10 blocks, which I think I sent you when I was hacking this offline.